samples/nrf_desktop/src/modules/ble_adv.c

```diff
@@ -52,6 +94,11 @@ static void ble_adv_start(void)
 
 	LOG_INF("Advertising started");
 
+	if (IS_ENABLED(CONFIG_DESKTOP_BLE_SWIFT_PAIR)) {
+		/* Switch to slower advertising after 30 seconds */
+		k_delayed_work_submit(&adv_param_update, K_SECONDS(30));
```

Comment:
You can make this parameter configurable (the 30s is the lower limit in the spec)

Reply:
Done